### PR TITLE
Delivery method

### PR DESCRIPTION
**Pull request by inescoelho**
Delivery method (#6) implementation

### CODE REVIEW

**Comment by joneslee85** [comment]
please rebase and squash into 1 commit for me

**Comment by rosafaria** [comment]
Do you want to merge this one first? Wouldn't it be better to merge the previous PR before this one? :)

File: lib/lotus/mailer/configuration.rb [hunk 1]
```diff
@@ -183,10 +182,65 @@ def copy!(base)
         end
       end
 
+      #Specify a global delivery method
```
Comment (joneslee85) [inline lib/lotus/mailer/configuration.rb]: nitpick, add a space after `#`

File: lib/lotus/mailer/configuration.rb [hunk 2]
```diff
@@ -183,10 +182,65 @@ def copy!(base)
...
+      #
+      # @return [Array] an array containing the delivery method and the optional settings as an Hash
+      #
+      # @since 0.1.0
+      #
+      # @example With Method only
+      # class DeliveryMethodSendmail
+      #   include Lotus::Mailer
+      #
+      #   Lotus::Mailer.configure do
```
Comment (joneslee85) [inline lib/lotus/mailer/configuration.rb]: the Mailer configuration should be not in the mailer class, it should be called separately:

```
class DeliveryMethodSendMail
  include Lotus::Mailer
end

Lotus::Mailer.confgure do
  delivery_method: sendmail
end
```

**Pull request merged**